config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -3053,26 +3058,9 @@ presubmits:
         securityContext:
           privileged: true
         volumeMounts:
-        - mountPath: /lib/modules
-          name: modules
-          readOnly: true
-        - mountPath: /sys/fs/cgroup
-          name: cgroup
         - mountPath: /etc/ssh-security
           name: ssh-security
-      dnsConfig:
-        options:
-        - name: ndots
-          value: "1"
       volumes:
-      - hostPath:
-          path: /lib/modules
-          type: Directory
-        name: modules
-      - hostPath:
-          path: /sys/fs/cgroup
-          type: Directory
-        name: cgroup
       - name: ssh-security
         secret:
           defaultMode: 256
```

Comment:
This file should be auto-generated and should not be part of the PR I guess.

Reply:
this one is checked in normally for :reasons: